oracle/helpers_test.go

```diff
@@ -196,14 +196,33 @@ var (
 		},
 	}
 )
+var _ oracle.PriceAggregator = &noOpPriceAggregator{}
+
+type noOpPriceAggregator struct{}
+
+func (n noOpPriceAggregator) SetProviderPrices(_ string, _ oracletypes.Prices) {
+}
+
+func (n noOpPriceAggregator) UpdateMarketMap(_ mmtypes.MarketMap) {
+}
+
+func (n noOpPriceAggregator) AggregatePrices() {
+}
+
+func (n noOpPriceAggregator) GetPrices() oracletypes.Prices {
+	return oracletypes.Prices{}
+}
+
+func (n noOpPriceAggregator) Reset() {
+}
```

Comment:
Might be useful to move this into the testutils directory.

Reply:
i tend to prefer to keep stuff like this in the package where its used, and only move it out when some other package might need it. happy to move it out if you disagree though